web-server/v0.4/components/Layout/Summary.js

```diff
@@ -16,22 +18,39 @@ class Summary extends React.Component {
     this.state = {
       summaryResult: [],
       iterations: [],
+      iterationSearch: [],
       columns: [],
-      loading: true
+      configData: {},
+      selectedConfig: [],
+      responseData: {},
+      selectedPort: "all",
+      ports: [],
+      loading: true,
+      searchText: '',
+      activeTab: 'iterations'
     }
   }
 
    componentDidMount() {
     this.setState({loading: true});
+    const { result, controller } = this.props;
     axios.get('http://es-perf44.perf.lab.eng.bos.redhat.com:9280/dsa.pbench.*/_search?source={ "query": { "match": { "run.name": "' + this.props.result + '" } }, "sort": "_index" }').then(res => {
       var result = [];
       result.push(res.data.hits.hits[0]);
       this.setState({summaryResult: result});
     });
 
-    axios.get('http://pbench.perf.lab.eng.bos.redhat.com/results/' + encodeURI(this.props.controller.slice(0, this.props.controller.indexOf("."))) +'/'+ encodeURI(this.props.result) + '/result.json').then(res => {
+    var iterationEndpoint = "";
+    if (controller.includes('.')) {
+      iterationEndpoint = 'http://pbench.perf.lab.eng.bos.redhat.com/results/' + encodeURI((controller).slice(0, (controller).indexOf("."))) +'/'+ encodeURI(result) + '/result.json';
```

Comment:
Can we find a way to remove this hard-coded `http://pbench.perf.lab.eng.bos.redhat.com` URL prefix?